library.rb

```diff
@@ -0,0 +1,22 @@
+class Person
```

Comment:
-  [ ] We should be able to retrieve the `id`, the `Getter` method should be defined like so:

`attr_reader :id`

Please kindly fix that.